src/App.js

```diff
@@ -1,16 +1,23 @@
 import "./App.css";
 import { MockBee } from "./backend/mockdocs/MockBee";
-import { Routes, Route, Link } from "react-router-dom";
+import { Routes, Route, Link, useLocation, Navigate } from "react-router-dom";
 import MockAPI from "./backend/mockdocs/MockMan";
-import { Cart, Home, Login, SignUp, Wishlist, ProductListing } from "./pages";
-import { useDocumentTitle } from "./customHooks/useDocumentTitle";
+import {
+	Cart,
+	Home,
+	Login,
+	SignUp,
+	Wishlist,
+	ProductListing,
+	NotFound,
+} from "./pages";
 import { Footer, Header } from "./components";
 
 function App() {
-	useDocumentTitle("MS | HOME");
+	const currentPath = useLocation().pathname;
 	return (
 		<div className="App body">
-			<Header />
+			{currentPath !== "/pagenotfound" && <Header currentPath={currentPath} />}
```

Comment:
Try to use the ternary operator instead of logical && when you are checking the condition and it's not returning true or false. You can refer to this article for a detailed explanation.  https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
NIce share, will update it.